### PR TITLE
Pulsechain

### PR DESCRIPTION
**Pull request by aggris2**
Fixes # .

Changes proposed in this pull request:
- PulseChain Testnet v2b update

### CODE REVIEW

File: src/networks.json [hunk 1]
```diff
@@ -737,6 +737,23 @@
     "start": 15123138,
     "imageIPFS": "QmYqkn8pJUaV9KcEPYEvRPwgbfeozLEvcQ9aEwKNRUL3cR"
   },
+  "941": {
+    "key": "941",
+    "name": "PulseChain Testnet V2B",
+    "shortName": "pulsechain",
+    "chainId": 941,
+    "network": "Testnet v2B",
+    "multicall": "0x959a437F1444DaDaC8aF997E71EAF0479c810267",
```
Comment (ChaituVR) [inline src/networks.json:746]: HI @aggris2 Can you verify and publish code here https://scan.v2b.testnet.pulsechain.com/address/0x959a437F1444DaDaC8aF997E71EAF0479c810267/contracts

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Revert changes from package-lock.json file

File: src/networks.json (hunk 1 above)
Comment (aggris2) [reply src/networks.json:746]: @ChaituVR done!

**Comment by ChaituVR** [comment]
Looks good except that there are changes in yarn.lock, once reverted this will be good to merge

**Pull request closed without merging**